src/index.tsx

```diff
@@ -38,6 +38,32 @@ dbRef
   });
 
 const initializeCourseViewer = (): void => {
+
+  // Read the cookie and check whether the given user is authorized
+  if (getCookieValue('user')) {
+    //There shouldn't be an error here, but to prevent unnecessary crashes or security issues a try catch here is useful
```

Comment:
Should be space 
```suggestion
    // There shouldn't be an error here, but to prevent unnecessary crashes or security issues a try catch here is useful
```